client/src/js/services/ModalService.js

```diff
@@ -279,4 +282,23 @@ function ModalService(Modal) {
       return instance.result;
     }
 
+    /** Find by Date interval modal */
+    function openDateInterval(request) {
+
+      var params = angular.extend(modalParameters, {
+        templateUrl  : 'partials/templates/modals/dateInterval.tmpl.html',
+        controller   : 'DateIntervalModalController',
+        controllerAs : '$ctrl',
+        size         : 'xs',
```

Comment:
![modal](https://cloud.githubusercontent.com/assets/5445251/16707886/11531150-4594-11e6-8d33-cbddfa4feb09.png)

Has a good look i think, and when the screen is so small we have : 

![modal2](https://cloud.githubusercontent.com/assets/5445251/16707889/36932536-4594-11e6-9a4f-c171d0a672c3.png)